### ISSUE TITLE
How can the program only run a window

### ISSUE THREAD

**Issue by yangyzp** [mpv-player/mpv#4299]
### mpv version and platform
0.24

How can the program only run a window？
How to configure the“mpv.conf”

**Comment by rossy** [comment]
> How can the program only run a window？

Could you rephrase this? I'm not sure what you mean.

If you want mpv to always open a window, add ``force-window`` to your mpv.conf.

If you want mpv to only ever create a single window, launch mpv with the [umpv][1] script (Linux only.)

[1]: https://github.com/mpv-player/mpv/raw/master/TOOLS/umpv

**Comment by yangyzp** [comment]
I want mpv to only ever create a single window, But my system is windows.~~~

**Comment by rossy** [comment]
Yeah, sorry. It should be possible to write something like umpv for Windows (in PowerShell maybe,) but no one has done it yet.

**Comment by qyot27** [comment]
Use the Run dialog to open stuff directly into mpv.exe?  If in newer versions of Windows, you could probably pin mpv.exe to the taskbar and drag-and-drop.

**Comment by yangyzp** [comment]
Thank you for your help.

**Comment by Skibicki** [comment]
#3811

**Comment by Skibicki** [comment]
Another [script](http://pgmacros.invisionzone.com/topic/7750-powershell-script-to-ensure-only-one-instance-of-mep-is-running/)
It runs properly from powershell.
```
if (-Not (Get-Process "mpv" -ErrorAction SilentlyContinue | Where-Object {-not $_.HasExited }))
{
  & "C:\mpv\mpv.exe"
}
```

**Comment by rossy** [comment]
Yep, this is a duplicate of #3811, closing.
